### DISCUSSION TITLE
[Discussions plugin] Option to hide categories

### DISCUSSION THREAD

**Discussion by Andre601** [lowlighter/metrics#742] [Ideas]
Loving the plugin for showing discussion related stats, but one thing I kinda dislike is the fact, that the categories listed can't be hidden/removed.

Because of this would I like to suggest such a feature to be implemented, since it could become quite a large SVG when responding to a lot of different categories.

**Comment by lowlighter** [comment]
Hello!
This will be implemented with #751 👍 
You'll be able to toggle discussions categories display with `plugin_discussions_categories` or limit the amount displayed with `plugin_discussions_categories_limit`

**Reply by Andre601** [reply]
Works perfectly. Thanks.

![image](https://user-images.githubusercontent.com/11576465/150366602-d05b96f2-027c-4020-9683-0f5b00b725af.png)
